ui-v2/app/services/lazy-proxy.js

```diff
@@ -0,0 +1,29 @@
+import Service from '@ember/service';
+import { get } from '@ember/object';
+
+export default Service.extend({
+  shouldProxy: function(content, method) {
+    return false;
+  },
+  init: function() {
+    this._super(...arguments);
+    const content = get(this, 'content');
+    for (let prop in content) {
+      (prop => {
```

Comment:
Why wrap this in an iife? Wouldn't removing the iife result in the same thing?

Reply:
Hmmm yeah , can't remember exactly. I think it's left over from a different version of that that had some async stuff in there somewhere. I vaguely remember putting this in as I hit that `for` async thing where the variable has already 'moved on', but I'm guessing that was in a previous version of this at some point. Will remove, thanks for spotting!